app/models/institution.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+class Institution < ActiveRecord::Base
+  EMPLOYER = 'ojt'
+  LOCALE = {
+    11 => 'city',
+    12 => 'city',
+    13 => 'city',
+    21 => 'suburban',
+    22 => 'suburban',
+    23 => 'suburban',
+    31 => 'town',
+    32 => 'town',
+    33 => 'town',
+    41 => 'rural',
+    42 => 'rural',
+    43 => 'rural'
+  }.freeze
+  DEGREES = {
+    0 => nil,
+    'ncd' => 'Certificate',
+    1 => 'Certificate',
+    '2-year' => 2,
+    2 => 2,
+    3 => 4,
+    4 => 4,
+    '4-year' => 4
+  }.freeze
+  TYPES = [
```

Comment:
Since school types are populated by the DS, and if we remove the validations (which may not be necessary - see below) then TYPES should not be required.